token/program-2022/src/extension/confidential_transfer_fee/processor.rs

```diff
@@ -367,41 +468,28 @@ pub(crate) fn process_instruction(
             )
         }
         ConfidentialTransferFeeInstruction::WithdrawWithheldTokensFromMint => {
-            msg!("ConfidentialTransferInstruction::WithdrawWithheldTokensFromMint");
-            #[cfg(all(feature = "zk-ops", feature = "proof-program"))]
```

Comment:
This removes the featurizing on `zk-ops`, so I think it'll fail to build with `zk-ops` disabled. I think we might need one more release using the feature off unfortunately, so could you restore that?

Reply:
Yes, sorry I missed this!